src/styles/getPopoverWithMeasuredContentStyles.ts

```diff
@@ -24,18 +17,11 @@ function computeHorizontalShift(anchorLeftEdge, menuWidth, windowWidth) {
 
     // Anchor is not in the gutter, so no need to shift it horizontally
     return 0;
-}
-
-/**
- * Compute the amount that the Context menu's Anchor needs to be vertically shifted
- * in order to keep it from displaying in the window.
- *
- * @param {Number} anchorTopEdge - Menu's anchor Top edge.
- * @param {Number} menuHeight - The height of the menu itself.
- * @param {Number} windowHeight - The height of the Window.
- * @returns {Number}
- */
-function computeVerticalShift(anchorTopEdge, menuHeight, windowHeight) {
+};
+
+// Compute the amount that the Context menu's Anchor needs to be vertically shifted
+// in order to keep it from displaying in the window.
+const computeVerticalShift = (anchorTopEdge: number, menuHeight: number, windowHeight: number): number => {
```

Comment:
```suggestion
function computeVerticalShift(anchorTopEdge: number, menuHeight: number, windowHeight: number): number {
```

NAB: In this case as you are typing the function directly, you can leave function signature.